### PR TITLE
fix(components/forms): allow toggle label to wrap

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#2336491](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2336491)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/777/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/777/storybooks/ag-grid-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/777/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/777/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/777/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/777/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/777/storybooks/popovers-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/777/storybooks/tabs-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/777/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/777/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/777/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/777/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/777/playground/)

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+  [ngClass]="{
+    'sky-toggle-switch-with-label': hasLabelComponent
+  }"
+>
   <button
     class="sky-toggle-switch-button"
     role="switch"
     type="button"
+    skyId
+    #toggle="skyId"
```
Comment (Blackbaud-SteveBrush) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html]: Template reference variables should be placed at the end of the tag.

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts [hunk 2]
```diff
@@ -102,12 +104,16 @@ export class SkyToggleSwitchComponent
...
   public hasLabelComponent = false;
+  public labelId: string;
 
   public enableIndicatorAnimation = false;
 
   @ContentChildren(SkyToggleSwitchLabelComponent)
   public labelComponents: QueryList<SkyToggleSwitchLabelComponent> | undefined;
 
+  @ViewChild('toggleLabel')
+  public toggleLabelEl: HTMLLabelElement | undefined;
```
Comment (Blackbaud-SteveBrush) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts]: What's this public property used for?

File: apps/code-examples/src/app/code-examples/forms/toggle-switch/inline-help/toggle-switch-demo.module.ts [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
 
 import { ToggleSwitchDemoComponent } from './toggle-switch-demo.component';
 
 @NgModule({
-  imports: [CommonModule, SkyHelpInlineModule, SkyToggleSwitchModule],
+  imports: [
+    CommonModule,
+    SkyHelpInlineModule,
+    SkyToggleSwitchModule,
+    FormsModule,
```
Comment (Blackbaud-SteveBrush) [inline apps/code-examples/src/app/code-examples/forms/toggle-switch/inline-help/toggle-switch-demo.module.ts]: I removed the `FormsModule` and the code example still worked. Do we need this for a different reason?

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html [hunk 4]
```diff
@@ -23,12 +31,16 @@
...
       >
     </div>
-    <!-- Non-breaking space needed to align switch to baseline! -->
-    <span *ngIf="hasLabelComponent" class="sky-toggle-switch-label">
-      <ng-container *ngTemplateOutlet="labelContent"></ng-container>
-    </span></button></span
-><span class="sky-control-help-container"
-  ><ng-content select=".sky-control-help"></ng-content
+  </button>
+  <span
```
Comment (Blackbaud-SteveBrush) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html:35]: Do we still need this `span` element?

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html (hunk 1 above)
Comment (johnhwhite) [reply libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html]: Moved.

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts (hunk 2 above)
Comment (johnhwhite) [reply libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts]: I didn't end up using it. Removed.

File: apps/code-examples/src/app/code-examples/forms/toggle-switch/inline-help/toggle-switch-demo.module.ts (hunk 3 above)
Comment (johnhwhite) [reply apps/code-examples/src/app/code-examples/forms/toggle-switch/inline-help/toggle-switch-demo.module.ts]: I was seeing an error on the preview site, but it looks like docstools provides `FormsModule` in StackBlitz so it wasn't broken there. I'm refactoring this to use a reactive form to simplify the example.

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html (hunk 4 above)
Comment (johnhwhite) [reply libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html:35]: That is what we're using to maintain the spacing that UX wants.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**